modules/Forum/pages/forum/edit.php

```diff
@@ -150,12 +150,12 @@
                     }
                     return $prev;
                 });
-                $accessible_labels = Forum::getAccessibleLabels($forum_labels, $user_groups);
+                $accessible_labels = Forum::getAccessibleLabels($forum_labels ?? [], $user_groups);
```

Comment:
Instead of adding `?? []` to here and line 158, it should be possible to specify `[]`  as the third parameter to array_reduce